scripts-available/CDB_CartodbfyTable.sql

```diff
@@ -649,3 +649,837 @@ BEGIN
   PERFORM cartodb.CDB_CartodbfyTable('public', reloid);
 END;
 $$ LANGUAGE PLPGSQL;
+
+
+-- -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+--
+--    NEW CARTODBFY CODE FROM HERE ON DOWN
+--
+-- -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+--
+--  CDB_CartodbfyTable2(reloid REGCLASS, destschema TEXT DEFAULT NULL)
+--    
+--     Main function, calls the following functions, with a little
+--     logic before the table re-write to avoid re-writing if the table
+--     already has all the necessary columns in place.
+--
+-- (1) _CDB_drop_triggers
+--     As before, this drops all the metadata and geom sync triggers
+--
+-- (2) _CDB_Has_Usable_Primary_ID()
+--     Returns TRUE if it can find a unique integer primary key named
+--    'cartodb_id' or can rename an existing key.
+--     Returns FALSE otherwise.
+--
+-- (3) _CDB_Has_Usable_Geom()
+--     Looks for existing EPSG:4326 and EPSG:3857 geometry columns, and
+--     renames them to the standard names if it can find them, returning TRUE.
```

Comment:
Is this the desired behaviour? Or do we want to keep the original columns?

I know it's a non-sense to have columns duplicated but as I use I can find myself lost because some of my columns disappeared.